docs/guides/aws-workspace.md

```diff
@@ -288,11 +288,13 @@ In [the next step](workspace-management.md), please use the following configurat
 
 ```hcl
 provider "databricks" {
-  host  = module.ai.databricks_host
-  token = module.ai.databricks_token
+  host  = databricks_mws_workspaces.this.workspace_url
```

Comment:
so, this is not correct and won't execute... this guide uses provider without an alias and lines 290-292 instantiate another provider without an alias. the point of `module.ai` was to explicitly enforce users to separate workspace creation and things within the workspace into two separate modules. please rewrite this section the way that it still enforces users to use multiple modules, but makes it clearer

Reply:
Hi serge, I have taken your comments in account and committed the changes. Pls let me know if it looks ok or needs further correction. Thanks